src/feed/api.ts

```diff
@@ -0,0 +1,25 @@
+import { Bee, Utils } from '@ethersphere/bee-js'
+import { Data } from '@ethersphere/bee-js/dist/src/types'
+import { bmtHashString, extractChunkData } from '../account/utils'
+import { getId } from './handler'
+import { bytesToHex } from '../utils/hex'
+import { keccak256Hash } from '../account/encryption'
+import { lookup } from './lookup/linear'
+import { Epoch } from './lookup/epoch'
+import Long from 'long'
+import { NoClue } from './lookup/lookup'
+
+const downloadTimeout = 1000
```

Comment:
it has to be an optional parameter (with this or a higher default) in the `getFeedData` function.